src/server_manager/web_app/karma.conf.js

```diff
@@ -0,0 +1,26 @@
+const webpackConfig = require('../test.webpack.js');
+
+module.exports = function(config) {
+  config.set({
+    frameworks: ['jasmine'],
+    files: [
+      '**.spec.ts',
+    ],
+    preprocessors: {
+      '**.spec.ts': ['webpack'],
+    },
+    reporters: ['progress'],
```

Comment:
There are some interesting third-party reporter packages. This is a basic one that shows you the name and exception for each failed test, but not much more.

The other standard built-in reporter is the one that just shows dots, which wasn't providing enough context.